packages/@aws-cdk/aws-rds/lib/serverless-cluster.ts

```diff
@@ -0,0 +1,544 @@
+/**
+ *  Properties to configure an Aurora Serverless Cluster
+ */
+export interface ServerlessClusterProps {
+  /**
+   * What kind of database to start
+   */
+  readonly engine: IClusterEngine;
+
+  /**
+   * Username and password for the administrative user
+   */
+  readonly masterUser: Login;
+
+  /**
+   * An optional identifier for the cluster
+   *
+   * @default - A name is automatically generated.
+   */
+  readonly clusterIdentifier?: string;
+
+  /**
+   * The number of days during which automatic DB snapshots are retained. Set
+   * to zero to disable backups.
+   *
+   * @default Duration.days(1)
+   */
+  readonly backupRetention?: Duration;
+
+  /**
+   * Name of a database which is automatically created inside the cluster
+   *
+   * @default - Database is not created in cluster.
+   */
+  readonly defaultDatabaseName?: string;
+
+  /**
+   * Indicates whether the DB cluster should have deletion protection enabled.
+   *
+   * @default - true if removalPolicy is RETAIN, false otherwise
+   */
+  readonly deletionProtection?: boolean;
+
+  /**
+   * Whether to enable the HTTP endpoint for an Aurora Serverless database cluster
+    *
+   * @default false
+   */
+  readonly enableHttpEndpoint?: boolean;
+
+  /**
+   * The VPC that this Aurora Serverless cluster has been created in.
+   *
+   */
+  readonly vpc: ec2.IVpc;
+
+  /**
+   * Where to place the instances within the VPC
+   *
+   * @default - the VPC default strategy if not specified.
+   */
+  readonly vpcSubnets?: ec2.SubnetSelection;
+
+  /**
+   * Scaling configuration
+   *
+   * @default - None
+   */
+  readonly scaling?: ServerlessScalingOptions;
+
+  /**
+   * The removal policy to apply when the cluster and its instances are removed
+   * from the stack or replaced during an update.
+   *
+   * @default - RemovalPolicy.SNAPSHOT (remove the cluster and instances, but retain a snapshot of the data)
+   */
+  readonly removalPolicy?: RemovalPolicy;
+
+  /**
+   * Security group.
+   *
+   * @default - a new security group is created.
+   */
+  readonly securityGroups?: ec2.ISecurityGroup[];
+
+  /**
+   * The KMS key for storage encryption.
+   *
+   * @default - the default master key will be used for storage encryption
+   */
+  readonly storageEncryptionKey?: kms.IKey;
+
+  /**
+   * Additional parameters to pass to the database engine
+   *
+   * @default - no parameter group.
+   */
+  readonly parameterGroup?: IParameterGroup;
+}
+
+/**
+ * Create a clustered database with a given number of instances.
+ */
+export interface IServerlessCluster extends IResource, ec2.IConnectable, secretsmanager.ISecretAttachmentTarget {
+  /**
+   * Identifier of the cluster
+   */
+  readonly clusterIdentifier: string;
+
+  /**
+   * The endpoint to use for read/write operations
+   * @attribute EndpointAddress,EndpointPort
+   */
+  readonly clusterEndpoint: Endpoint;
+
+  /**
+   * Endpoint to use for load-balanced read-only operations.
+   * @attribute ReadEndpointAddress
```

Comment:
No port here?

Reply:
no port for the reader endpoint (same as for `Cluster`)